addons/website_blog/tests/test_ui.py

```diff
@@ -65,3 +65,24 @@ def test_avatar_comment(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.headers.get('Content-Type'), 'image/png')
         self.assertRegex(response.headers.get('Content-Disposition', ''), r'mail_message-\d+-author_avatar\.png')
+
+    def test_blog_access_rights(self):
+        group_website_blog_manager_id = self.ref("website_blog.group_website_blog_manager")
+        group_web_designer_id = self.ref("website.group_website_designer")
+        group_employee_id = self.ref("base.group_user")
+        self.user_with_blog_manager_rights = self.env["res.users"].with_context({"no_reset_password": True}).create({
+            "name": "Adam Blog Manager",
+            "login": "adam",
+            "email": "adam.manager@example.com",
+            "notification_type": "email",
```

Comment:
Do we need this?

Reply:
You're right! After reviewing, I found that ` "notification_type": "email" ` not that necessary. We'll make sure to improve/remove this.
Thanks for pointing it out!